x-pack/plugins/cloud_security_posture/public/pages/findings/translations.ts

```diff
@@ -6,142 +6,187 @@
  */
```

Comment:
consider adding sub sections for `findingFlyout` and `<x>Tab`
so for example:
`xpack.csp.findings.findingFlyout.remediationTab.nameLabel`